src/sentry/web/urls.py

```diff
@@ -49,21 +49,6 @@
 __all__ = ("urlpatterns",)
 
 
-def init_all_applications():
```

Comment:
Why'd we remove this? I suppose it's probably not necessary since we register these as `INSTALLED_APPS`?

Reply:
Yep, probably not necessary cause we now have the django.setup postcondition that the django app registry is ready. I'm hoping this doesn't break anything beyond test coverage, haha.

Reply:
Does not removing it cause problems in 1.9?

Reply:
It's the get_apps and get_models that have to be ported over. I started on it (had some trouble with one or the other, I forget), then realized it can probably just be removed, haha.

Reply:
Ok, I think I agree. All this does is look at all installed apps and import the models. But every django installed app will have done this already...

Slightly scared, but I think this will be ok. This code is also from 2012, lol.